contracts/dao/neutron-staking-tracker/src/contract.rs

```diff
@@ -208,8 +208,8 @@ pub(crate) fn after_validator_created(
 pub(crate) fn after_validator_removed(
     deps: DepsMut,
     env: Env,
-    valoper_address: String,
     _valcons_address: String,
+    valoper_address: String,
 ) -> Result<Response, ContractError> {
```

Comment:
since we don't use cons addr at all, why don't we just remove it from the func parameters? why keep that unneeded field, add confusion and have mistake-prone code? here and in all other hook handlers with cons_addr, like that:
<img width="1124" alt="image" src="https://github.com/user-attachments/assets/18504ca3-9235-4e28-b7bc-6618ae42e0bf" />